### ISSUE TITLE
Consider backtick as multiline string indicator?

### ISSUE THREAD

**Issue by prideofconcept** [emilk/Configuru#4]
> """ starts a verbatim multi-line string

Should a ` (backtick) also illicit this behaviour, a la ES6

**Comment by emilk** [comment]
Yes, maybe it should! However, I don't have the time to add this feature, but as they say: "PR:s welcome" :)
